### PR TITLE
docs: Kingfisher Views; add logging

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-views/issues/93

### CODE REVIEW

**Comment by odscjames** [comment]
[Store outputs of add-view, refresh-views and field-counts; migrate away from logzero](https://trello.com/c/T7uXC7kD/208-store-outputs-of-add-view-refresh-views-and-field-counts-migrate-away-from-logzero)

File: docs/use/kingfisher-views.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+----------------------------------------
+
+This is the same as for Kingfisher Process - :ref:`please read this documentation<connect-process-server>`.
+
+Review log files
+----------------
+
+This is the same as for Kingfisher Process - :ref:`please read this documentation<kingfisher-process-review-log-files>`.
+
+For more information, read Kingfisher View' `logging documentation <https://kingfisher-views.readthedocs.io/en/latest/logging.html>`__.
```
Comment (jpmckinney) [inline docs/use/kingfisher-views.rst:16]: Replace with:

```rst
See the :doc:`kingfisher-process` page to connect to the server and review log files.

For more information on the format of log messages, read Kingfisher Views' `logging documentation <https://kingfisher-views.readthedocs.io/en/latest/logging.html>`__.

**Comment by odscjames** [comment]
Server config updated, will update docs P.R. shortly.

**Comment by odscjames** [comment]
I edited slightly; the format is actually the same, it's the topic/name which will be different.

**Comment by jpmckinney** [comment]
Thanks!

**Pull request merged**